components/AlphabetGlossary.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <div class="text-right mb-5 mr-[10%]">
+      <button
+        @click="expandAll"
```

Comment:
Is it possible to add animations when the sections expand out? Don't worry if it is too much effort. Only do it if it is reasonably quick to achive. Otherwise we can add this as an issue to look at at a later date. Thanks Ruby

Reply:
I have no experience with animations, so think would take me too much time for now but would be nice addition. Thank you.

Reply:
No worries. I'll add an issue for this and mark this as resolved.